src/js/components/LeftPane.js

```diff
@@ -1,38 +1,123 @@
 /* @flow */
 
 import {useDispatch, useSelector} from "react-redux"
-import React, {useState} from "react"
-import classNames from "classnames"
-
-import {LinkButton} from "./Typography"
+import React, {useState, type Node} from "react"
 import {XLeftPaneCollapser} from "./LeftPaneCollapser"
 import {XLeftPaneExpander} from "./LeftPaneExpander"
-import {globalDispatch} from "../state/GlobalContext"
 import FilterTree from "./FilterTree"
-import Investigation from "../state/Investigation"
 import InvestigationLinear from "./Investigation/InvestigationLinear"
-import Pane, {PaneHeader, PaneTitle, Left, Right, Center} from "./Pane"
-import View from "../state/View"
+import Pane from "./Pane"
+import Layout from "../state/Layout"
+import BookIcon from "../icons/BookSvgIcon"
+import styled from "styled-components"
+import usePopupMenu from "./hooks/usePopupMenu"
+import DropdownArrow from "../icons/DropdownArrow"
 
-export function LeftPane() {
-  let [showCollapse, setShowCollapse] = useState(true)
-  let view = useSelector(View.getInvestigationView)
-  let isOpen = useSelector(View.getLeftSidebarIsOpen)
-  let width = useSelector(View.getLeftSidebarWidth)
-  let dispatch = useDispatch()
+const Arrow = (props) => {
+  return (
+    <svg className={props.className} onClick={props.onClick} viewBox="0 0 8 8">
+      <polygon points="0 0 8 4 0 8" />
+    </svg>
+  )
+}
 
-  function onDrag(e: MouseEvent) {
-    const width = e.clientX
-    const max = global.innerWidth
-    dispatch(View.setLeftSidebarWidth(Math.min(width, max)))
+const SectionHeader = styled.div`
+  display: flex;
+  flex-direction: row;
+  background-color: ${(props) => props.theme.colors.coconut};
+  height: 32px;
```

Comment:
The height for this is 24px. (since the box-sizing is "border-box", the two 1px borders will be subtracted from this height in actuality. So when you inspect the element in chrome, you should see:

<img width="170" alt="image" src="https://user-images.githubusercontent.com/3460638/84090128-ad468780-a9a5-11ea-9f6d-cb412ba23c8f.png">